### ISSUE TITLE
Hot reloading is not working on windows docker desktop

### ISSUE THREAD

**Issue by rajeshpatil74** [pnp/docker-spfx#76]
I am using docker image from - https://hub.docker.com/r/waldekm/spfx/tags and with latest image on windows docker desktop , hot reloading is not working (changing code doesnt refresh the webpart in browser)
Issues are - 
1. The webpart is not getting loaded using localhost. - 1.13.1 not supporting localhost. So its fine.
2.  Webpart gets loaded on sharepoint site workbench, but with manual refresh changes are not reflected.

With docker image 18 months old  (docker pull waldekm/spfx:1.11.0.) this was working.
  - Webpart gets loaded in localhost and changes loads it automatically.
  - On sharepoint site, manual refresh is needed. --> **this should work with latest docker image. As of now one need to terminate gulp command and reissue it.**

Even tried this link - https://spblog.net/post/2020/12/22/sharepoint-framework-with-eslint.
**Whenever code changed recompile not gets triggered and page will have old content.**

**Comment by necik11** [comment]
I don't know if I understood correctly and maybe it's a stupid question, but did I try to wait a while longer? 😄 
I wrote about it here:
https://github.com/pnp/docker-spfx/issues/70#issuecomment-1026146326
"I think the change took effect and after: Finished subtask 'webpack' after 28 s"
And only after this will the reload be called automatically ...
If I misunderstood, I apologize :)

**Comment by shurick81** [comment]
Is this a duplicate of #69?

**Comment by waldekmastykarz** [comment]
Closing due to inactivity
